src/Pages/ProductDetail/ProductDetail.jsx

```diff
@@ -104,14 +105,15 @@ class ProductDetail extends React.Component{
                         <span>home - {productData.product_menu} - {productData.product_category}</span>
                     </div>
                </div>
-                <ImgPurchInfo productName={productData &&  productData.product_name} imgUrl={productData && productData.image} id={productData && productData.id} price={productData && productData.price} reviewArray={reviewList&& reviewList.map((el)=>el.review_rate)}/>
+                <ImgPurchInfo productName={productData &&  productData.product_name} imgUrl={productData && productData.image} id={productData && productData.id} price={productData && productData.price} reviewArray={reviewList&& reviewList.map((el)=>el.rate)}/>
```

Comment:
아래와 같이 하면 더 깔끔해지겠네요. 
prettier가 안깔려 있나요?! 가독성이 좋지 않아요. 코드 정리가 필요해보입니다!
```suggestion
                { productData && 
                <ImgPurchInfo 
                productName={productData.product_name} 
                imgUrl={ productData.image} 
                id={productData.id} 
                price={productData.price} 
                reviewArray={reviewList.map((el)=>el.rate)}/>
                }
```

Reply:
제가 어제 뭘 만졋는지 모르겟지만 파일이 다 날아가면서 vs코드에 받아논것도 다 날아갓나봐요.. 인덴트도 설정해놧엇는데.. 다시 해놓겟습니다....